src/operator/src/statement.rs

```diff
@@ -171,9 +171,15 @@ impl StatementExecutor {
                 self.create_flow(stmt, query_ctx).await?;
                 Ok(Output::new_with_affected_rows(0))
             }
-            Statement::DropFlow(_stmt) => {
-                // TODO(weny): implement it.
-                unimplemented!()
+            Statement::DropFlow(stmt) => {
+                self.drop_flow(
+                    query_ctx.current_catalog().to_string(),
+                    format_raw_object_name(stmt.flow_name()),
+                    stmt.drop_if_exists(),
+                    query_ctx,
+                )
+                .await?;
+                Ok(Output::new_with_affected_rows(0))
```

Comment:
```suggestion
                self.drop_flow(
                    query_ctx.current_catalog().to_string(),
                    format_raw_object_name(stmt.flow_name()),
                    stmt.drop_if_exists(),
                    query_ctx,
                )
                .await
```

Same as  `create flow`